<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QScrollBar" name="horizontalScrollBar">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="zoomIn_but">
      <property name="text">
       <string>放大</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="zoomOut_but">
      <property name="text">
       <string>缩小</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="reset_but">
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>波形图/语谱图</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCustomPlot" name="waveform_wid_1" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="readwav_act"/>
    <addaction name="play_act"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>新建</string>
    </property>
    <addaction name="newsinglewav_act"/>
    <addaction name="newdoublewav_act"/>
   </widget>
   <widget class="QMenu" name="newWindow_menu">
    <property name="title">
     <string>新界面</string>
    </property>
    <addaction name="windowopen"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="spectrumwind_open_act"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>频率响应</string>
    </property>
    <addaction name="freqResp_act"/>
    <addaction name="winFunc_act"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="newWindow_menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="play_act">
   <property name="text">
    <string>播放</string>
   </property>
  </action>
  <action name="periodwind_open_act">
   <property name="text">
    <string>周期分析</string>
   </property>
  </action>
  <action name="spectrumwind_open_act">
   <property name="text">
    <string>频谱分析</string>
   </property>
  </action>
  <action name="filterwind_open_act">
   <property name="text">
    <string>滤波</string>
   </property>
  </action>
  <action name="readwav_act">
   <property name="text">
    <string>从文件读取（.wav）</string>
   </property>
  </action>
  <action name="newsinglewav_act">
   <property name="text">
    <string>录制单声道</string>
   </property>
  </action>
  <action name="newdoublewav_act">
   <property name="text">
    <string>录制双声道</string>
   </property>
  </action>
  <action name="windowopen">
   <property name="text">
    <string>open</string>
   </property>
  </action>
  <action name="spectrogram_act">
   <property name="text">
    <string>语谱分析</string>
   </property>
  </action>
  <action name="freqResp_act">
   <property name="text">
    <string>频率响应</string>
   </property>
  </action>
  <action name="winFunc_act">
   <property name="text">
    <string>窗口函数</string>
   </property>
  </action>
  <action name="actionp">
   <property name="text">
    <string>频谱分析</string>
   </property>
  </action>
  <action name="actionp_2">
   <property name="text">
    <string>语谱分析</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
